src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Patterns.cs

```diff
@@ -48,12 +48,16 @@ BoundExpression TranslatePattern(BoundExpression input, BoundPattern pattern)
                         input = _factory.Local(temp);
                         for (int i = 0; i < pat.Subpatterns.Length; i++)
                         {
-                            var subProperty = pat.Subpatterns[i].Property;
+                            var subProperty = (pat.Subpatterns[i].Member as BoundPropertyPatternMember)?.MemberSymbol;
```

Comment:
Let's not write code for a feature we don't have a basic design for and for which we haven't even discussed an implementation strategy. This is also inconsistent with our fail-fast policy for compiler bugs. Please change this code to a "normal" cast and remove null-conditional access off of ```subProperty``` below.